py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -245,7 +245,40 @@ def aten_ops_fmod(
     return impl.elementwise.fmod(ctx, target, SourceIR.ATEN, name, args[0], args[1])
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.relu.default)  # type: ignore[misc]
```

Comment:
Undo this change

Reply:
Oh I think this got deleted in the formatting change, it is present below

Reply:
I just intended to have the `ignore` be kept